arviz/tests/base_tests/test_plots_matplotlib.py

```diff
@@ -1441,3 +1442,88 @@ def test_plot_bpv_discrete():
     fake_model = from_dict(posterior_predictive=fake_pp, observed_data=fake_obs)
     axes = plot_bpv(fake_model)
     assert not isinstance(axes, np.ndarray)
+
+
+@pytest.mark.parametrize(
+    "kwargs",
+    [
+        {},
+        {"y_hat": "bad_name"},
+        {"x": "x1"},
+        {"x": ("x1", "x2")},
+        {
+            "x": ("x1", "x2"),
+            "y_kwargs": {"color": "blue", "marker": "^"},
+            "y_hat_plot_kwargs": {"color": "cyan"},
+        },
+        {"x": ("x1", "x2"), "y_model_plot_kwargs": {"color": "red"}},
+        {
+            "x": ("x1", "x2"),
+            "kind_pp": "hdi",
+            "kind_model": "hdi",
+            "y_model_fill_kwargs": {"color": "red"},
+            "y_hat_fill_kwargs": {"color": "cyan"},
+        },
+    ],
+)
+def test_plot_lm(models, kwargs):
+    idata = models.model_1
+    if "constant_data" not in idata.groups():
+        y = idata.observed_data["y"]
+        x1data = y.coords[y.dims[0]]
+        idata.add_groups({"constant_data": {"_": x1data}})
+        idata.constant_data["x1"] = x1data
+        idata.constant_data["x2"] = x1data
+
+    axes = plot_lm(idata=idata, y="y", y_model="eta", xjitter=True, **kwargs)
+    assert np.all(axes)
+
+
+def test_plot_lm_multidim(multidim_models):
+    idata = multidim_models.model_1
+    axes = plot_lm(
+        idata=idata,
+        x=idata.observed_data["y"].coords["dim1"].values,
+        y="y",
+        plot_dim="dim1",
+        show=True,
+        figsize=(4, 16),
+    )
+    assert np.all(axes)
+
+
+def test_plot_lm_bad_kind(models, multidim_models):
+    idata1 = models.model_1
```

Comment:
This one isnt your fault, but because we split our tests by plotting backend we get tons of repeated code. No need to change here. @ahartikainen  curious what your thoughts are

Reply:
Didn't get your point here. Confused what's not my fault 😆

Reply:
Theres a lot of duplication of code between `test_plots_matplotlib`, and the bokeh equivalent. Ideally we would not want to repeat things like the parameterization etc etc, and instead make the backend itself a parameter

Reply:
So does that mean, testing for both backends in a single file?

Reply:
Removed `test_plot_lm_bad_kwargs` from bokeh tests.